src/pages/docs/infrastructure/workers/dynamic-worker-pools.md

```diff
@@ -28,26 +28,26 @@ Each worker is provisioned exclusively to a specific customer, and is completely
 
 ## Dynamic Worker Images
 
-Each dynamic worker pool can specify the worker image used. Windows Server Core 2019 is the default. Ubuntu Server 22.04 worker images are also available.
+Each dynamic worker pool can specify the worker image used. Ubuntu Linux 22.04 is the default. Windows Server Core 2019 worker images are also available.
```

Comment:
In this PR we aim to simply swap the OS emphasis.

For context, this sentence is a little misleading since there are two things going on here: the **default image** version for each OS (Ubuntu -> 22.04), and then which Worker Pool is the initial default (and by extension, which image is used by the default pool). However, we're hoping to address this as part of a broader overhaul in #2025 .